plumpy/processes.py

```diff
@@ -81,14 +80,14 @@ def ensure_not_closed(func):
 
     @functools.wraps(func)
     def func_wrapper(self, *args, **kwargs):
-        if self._closed:
-            raise exceptions.ClosedError("Process is closed")
+        if self._closed:  # pylint: disable=protected-access
+            raise exceptions.ClosedError('Process is closed')
         return func(self, *args, **kwargs)
 
     return func_wrapper
 
 
-@persistence.auto_persist('_pid', '_CREATION_TIME', '_future', '_paused', '_status', '_pre_paused_status')
+@persistence.auto_persist('_pid', '_creation_time', '_future', '_paused', '_status', '_pre_paused_status')
```

Comment:
Might need a little care here, just 'cause this may make it incompatible with things that have the caputalised '_CREATION_TIME', obviously this would only affect users at the 'upgrade boundary'.  On way around this would be to check for the capitalised version manually in `load_instance_state`.

Reply:
yeah, I thought about this and how it will invalidate existing checkpoints, however, since anyway this will go in a new minor release and so will also require a new release from AiiDA, this is fine. AiiDA does not guarantee that live processes will continue to work after an update. Users will have to finish running their processes and stop the daemon before upgrading the installation. So I think it is fine

Reply:
Roger!